app/kumactl/cmd/install/render_helm_files.go

```diff
@@ -78,10 +110,10 @@ func renderHelmFiles(
 
 func loadCharts(templates []data.File) (*chart.Chart, error) {
 	files := []*loader.BufferedFile{}
-	for _, f := range templates {
+	for _, template := range filterHelmTemplates(templates) {
```

Comment:
I think that we should take a consistent approach to filtering the CRD files. In `install_crds.go` we filter at the top level before any subsequent processing, so why don't we just do the same thing for installing the control plane?